torch/_inductor/standalone_compile.py

```diff
@@ -62,6 +62,75 @@ def __init__(
     def __call__(self, *args: Any) -> Any:
         return self._compiled_fn(*args)
 
+    def to_bytes(self) -> bytes:
```

Comment:
I think in the long term we don't want to use standalone_compile in vllm, but fine to do for now I guess. 

Could we have some basic unittest to verify to_bytes() and from_bytes() and finish a save/load roundtrip?